docs/configuration.md

```diff
@@ -169,5 +169,108 @@ php artisan up
 ```
 > {tip} You may customize the default maintenance mode template by defining your own template at `resources/views/errors/503.blade.php`.
 
+## Configuration options
+
+|Name|Description|Default|
+|---|---|---|
+|`APP_NAME`|The gallery name|`Lychee`|
+|`APP_ENV`|Environment of your gallery, `production` or `development`|`production`|
+|`APP_DEBUG`|Enable debug mode|`false`|
+|`APP_URL`|The URL of your gallery|`http://localhost`|
+|`APP_KEY`|Your app key which is used for encryption (set during installation)|`null`|
+|`TIMEZONE`|The timezone of your server|system timezone of server|
```

Comment:
Shouldn't the description of this one be more like "The timezone of your photos"?